create_framework/http-foundation.rst

```diff
@@ -121,6 +121,15 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    // composer.json
```

Comment:
JSON does not have comments. So I think we should be avoid such constructs here.